components/navbar/Navbar.tsx

```diff
@@ -0,0 +1,16 @@
+import React from "react";
+import Link from "next/link";
+
+type PropTypes = {};
+
+const Navbar: React.FC<PropTypes> = () => {
+  return (
+    <nav className="flex justify-between">
```

Comment:
How about making navbar a HOC so that whenever we use it we can just wrap actual content inside it without worrying about its position?